etc/containerpilot.json5

```diff
@@ -0,0 +1,34 @@
+{
+  consul: "127.0.0.1:8500",
+  jobs: [
+    {
+      name: "preStart",
+      exec: ["/usr/local/bin/consul-manage", "preStart"],
+    },
+    {
+      name: "consul",
+      port: 8500,
+      exec: ["/bin/consul", "agent", "-server",
+             "-bootstrap-expect", "3",
```

Comment:
Fair enough. We can't put the exec in the Compose manifest anymore, but we can do something with the env vars.